openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/data_insight.py

```diff
@@ -37,9 +47,47 @@ def build_data_insight_workflow_config(
     Given an airflow_pipeline, prepare the workflow config JSON
     """
 
+    try:
+        metadata = OpenMetadata(config=ingestion_pipeline.openMetadataServerConnection)
+    except Exception as exc:
+        raise ClientInitializationError(f"Failed to initialize the client: {exc}")
+
+    openmetadata_service = metadata.get_by_name(
+        entity=MetadataService, fqn=OPENMETADATA_SERVICE_FQN
```

Comment:
Hi @TeddyCr actually I moved this out. Might be cleaner to pick up the name from the `ingestion_pipeline.service.fullyQualifiedName`

Reply:
I'll update it